spec/spec_helper.rb

```diff
@@ -43,8 +43,14 @@ def reset_all
   def with_unfulfilled_double
     d = double("double")
     yield d
+  ensure
     reset d
   end
+
+  def expect_fast_failure_from(double, class_or_message=Exception, message=/.*/, &blk)
```

Comment:
Actually, I suspect that there is no need to specify a class other than `RSpec::Mocks::MockExpectationError`.  If that's the case, this can be simplified to:

``` ruby
def expect_fast_failure_from(double, *fail_with_args)
  expect { yield double }.to fail_with(*fail_with_args)
  reset double
end
```